lib/usermanager.js

```diff
@@ -832,6 +832,37 @@ exports = module.exports = {
       });
     });
 
+    rest.post('/user/invite', function(req, res) {
+      var user = self.getCurrentUser();
+
+      self.hasPermission('create', user, '/api/user/invite', function(err, isAllowed) {
+        if (err) {
+          res.statusCode = 500;
+          return res.json(err);
+        }
+        if (!isAllowed) {
+          res.statusCode = 401;
+          return res.json({success: false, message: 'Access denied'});
+        }
+        self.retrieveUser({ _id: user._id }, { }, function (err, userObject) {
+          if (err) {
+            res.statusCode = 500;
+            return res.json(err);
+          }
+          var inviterDetails = userObject.email;
+          if (userObject.firstName && userObject.lastName) {
+            inviterDetails = userObject.firstName + ' ' + userObject.lastName + ' (' + inviterDetails + ')';
+          }
+          var subject = app.polyglot.t('app.emailinvitesubject');
+          var body = app.polyglot.t('app.emailinvitebody', { inviter: inviterDetails, rootUrl: configuration.getConfig('rootUrl'), resetUrl: configuration.getConfig('rootUrl') + '/#user/forgot' });
+          app.mailer.send(req.body.email, subject, body, { name: 'emails/invite.hbs' }, function() {
+            res.statusCode = 200;
```

Comment:
Please can we add in error handling here as per:
https://github.com/adaptlearning/adapt_authoring/blob/3f060704f7a94bf3d19f1f69f26c60cdfc5e2fa8/plugins/auth/local/index.js#L286-L288

Reply:
:+1: sorry I missed it